src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,22 @@
+package wolox.training.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class BookController {
+
+    @GetMapping("/greeting")
+    public String greeting(
+        @RequestParam(name = "name", required = false, defaultValue = "Mundo") String name,
```

Comment:
As a general comment, try to be consistent with the wordings, for example if the message is `Hello, name`, the default value should be "world" rather than "Mundo" :)